### PR TITLE
Centralized services page

### PR DESCRIPTION
**Pull request by maxtan84**
Centralized all Math CnD, Board Games, Locker Sign outs, and MC Comfy into one services page.

### CODE REVIEW

File: views/pages/services/student-services.pug
Comment (some-molasses) [inline views/pages/services/student-services.pug:14]: Could you use `section` tags for each `div.service`?  It would help for accessibility!

File: views/pages/services/student-services.pug
Comment (some-molasses) [inline views/pages/services/student-services.pug:42]: Two things:
* Can we add indentation so that the entire text is included as part of the `p` tag?
* Is there a reason not to use pug for the `a` tag?  We should try to use pug where possible!

```suggestion
            p The Store Manager Rose Penner can be reached at: 519-888-4567 ext. 36373 or at 
              | a(href="mailto:r2penner@uwaterloo.ca") r2penner@uwaterloo.ca. 
              | The CnD can be reached directly at 519-888-4567 ext. 36521.    
```

File: views/pages/services/student-services.pug
Comment (some-molasses) [inline views/pages/services/student-services.pug:27]: This link seems to be broken 😔 

remove?

**Review by some-molasses — COMMENTED** [review]
love the design!

File: views/pages/services/student-services.pug
Comment (ad-world) [inline views/pages/services/student-services.pug:1]: could you remove all the copy from this page and put it into a data file in the `server/data` dir? that way the content on this page can be updated through the cms, instead of having to come into this file and change pug markdown

**Review by some-molasses — APPROVED** [review]

**Review by some-molasses — COMMENTED** [review]
approved pending Aryaman's approval

**Comment by ad-world** [comment]
i think there's still some design changes needed, but since we are redesigning the pages it's probably not worth putting them in this pr since they're gonna be changed anyway. approving and merging for now. thanks!

**Review by ad-world — APPROVED** [review]

**Pull request merged**
